modules/apps/headless/headless-admin-user/headless-admin-user-test/src/testIntegration/java/com/liferay/headless/admin/user/resource/v1_0/test/AccountResourceTest.java

```diff
@@ -533,6 +619,35 @@ protected String testGetOrganizationAccountsPage_getOrganizationId()
 		return String.valueOf(organization.getOrganizationId());
 	}
 
+	@Override
+	protected Account
+			testGetOrganizationByExternalReferenceCodeAccountsPage_addAccount(
+				String externalReferenceCode, Account account)
+		throws Exception {
+
+		Account putAccount = accountResource.putAccountByExternalReferenceCode(
+			account.getExternalReferenceCode(), account);
+
+		Organization organization =
+			_organizationLocalService.fetchOrganizationByExternalReferenceCode(
+				externalReferenceCode, TestPropsValues.getCompanyId());
+
+		accountResource.postOrganizationAccounts(
```

Comment:
why we cannot use the postOrganizationByExternalReferenceCodeAccounts?

Reply:
@smottal there is no such method, I could use postOrganizationAccountsByExternalReferenceCode or postOrganizationByExternalReferenceCodeAccountsByExternalReferenceCode.
The latter is better I think, because in that case I don't have to fetch the org separately.